pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 79
+target-version = ['py37', 'py38']
```

Comment:
Can we also disable string normalization?  Primarily because we want black to leave string prefixes alone. `r` and `R` mean different things for MagicPython.

Reply:
We can, sure. Care to elaborate how they're different? Python doesn't treat them differently.

Reply:
MagicPython highlights `r"foo"` as a regexp, and `R"foo"` as an uninterpreted raw string, since the majority of uses of raw strings are regexps.

Reply:
It it's possible to tell black to only normalize the quotes, and not prefixes, that's fine too.